app/controllers/admin/import_ect/find_ect_controller.rb

```diff
@@ -29,7 +29,7 @@ def create
       # Already registered
       rescue Admin::Errors::TeacherAlreadyExists
         existing_teacher = Teacher.find_by(trn: @pending_induction_submission.trn)
-        redirect_to admin_teacher_path(existing_teacher), notice: "Teacher #{existing_teacher.trn} already exists in the system"
+        redirect_to admin_teacher_induction_path(existing_teacher), notice: "Teacher #{existing_teacher.trn} already exists in the system"
```

Comment:
I'm just flagging that once this whole change is completed that this would actually redirect to the admin_teacher_path and show the overview tab. I'm assuming that for the moment this is fine and will be addressed in your next PR.

Reply:
Yup, @peteryates has got us covered. From the ticket: 

> We can line a PR up for post launch that fixes the ordering and links so the overview page becomes the main one.